i18n/config.js

```diff
@@ -15,7 +15,7 @@ i18next
       const isDirectory = lstatSync(joinedPath).isDirectory()
       return isDirectory
     }),
-    ns: ['translations', 'trending', 'meta', 'links'],
+    ns: ['translations', 'trending', 'meta-tags', 'links'],
```

Comment:
why there is dash ( `-` ) in the middle?

Reply:
The dash is there because meta tags is two words, so it makes sense.

Also, one of the goals with this PR is to bring the localization a bit closer to what we have in the main repo, and the name of the file we use there to set the attributes for some meta tags is named `meta-tags.json`.